install_trivy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Seems very specific to an OS? Not available through apt?

Reply:
It isnt :( the great thing about this approach is that I can directly get a DEB package from the repo this way, really giving you the latest version

Reply:
Yeah trivy seems evil when it comes to installation. Not sure I like any of the options tbh. E.g. what if a consumer is on macos or windows. This solution works only for debian and downstream 
https://aquasecurity.github.io/trivy/v0.18.3/installation/#install-script

Reply:
Shall we ditch Trivy then? I mean semgrep should be similar..

Reply:
(the reason for me saying this, is that I am not sure whether this is still as pythonic as we would like it to be if we cant install all dependencies with pip)

Reply:
Yeah can we do that in the meantime? Can always have a play around and see about adding back at a later date :)